### PR TITLE
Add health check service

### PR DESCRIPTION
**Pull request by isSerge**
Implementation of `health-check` service to let the [Status page](https://github.com/subspace/status) query health status of Postgres and internal services using Prometheus (`ingest` and `processor`)

Related to #203

### CODE REVIEW

**Review by natachadelarosa — DISMISSED** [review]
LGTM!

**Review by natachadelarosa — APPROVED** [review]
LGTM!

File: health-check/src/pg.ts [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  }
+}
+
+const server = createServer(async (req, res) => {
+  const authHeader = req.headers.authorization;
+
+  if (authHeader && authHeader.startsWith('Bearer ')) {
+    const token = authHeader.split('Bearer ')[1];
+
+    if (token === process.env.SECRET) {
```
Comment (nazar-pc) [inline health-check/src/pg.ts:45]: This is very bad and easy to guess. Never check secrets this way, you need to use constant-time comparison.

https://en.wikipedia.org/wiki/Timing_attack

Comment (isSerge) [reply health-check/src/pg.ts:45]: I would not use this approach for another service, but this is a pretty basic service and I just wanted to add basic protection

Comment (nazar-pc) [reply health-check/src/pg.ts:45]: I mean if you add secret check, add it properly. I don't recall if JS already has built-in function for that, but there are definitely some packages that do that.

Comment (isSerge) [reply health-check/src/pg.ts:45]: this can be done natively using `crypto.timingSafeEqual`, I will submit a PR

**Pull request merged**